lib/astutil.js

```diff
@@ -196,6 +196,17 @@ ast.Tex.defineVisitor("contains_func", {
             match(target, '\\end{'+t+'}') ||
             matrix_has(m);
     },
+    MULTLINE: function(target, t, m) {
```

Comment:
what is the difference in the functions for multine and matrix. Can't you just create one functaion and assign it to both keys MATRIX and MULTILINE